Client/Frontend/Browser/BrowserLocationView.swift

```diff
@@ -50,6 +50,8 @@ class BrowserLocationView : UIView, UIGestureRecognizerDelegate {
         stopReloadButton = UIButton()
         stopReloadButton.setImage(ImageReload, forState: .Normal)
         stopReloadButton.addTarget(self, action: "SELtapStopReload", forControlEvents: .TouchUpInside)
+        stopReloadButton.isAccessibilityElement = true
```

Comment:
Since stop `stopReloadButton` is a `UIButton`, I think this line is unnecessary.